deployment/helm/synopsys-alert/README.md

```diff
@@ -452,6 +455,46 @@ For the on-premise database deployment a second Persistent Volume must be create
     databaseName: <DATABASE_NAME>
   ```
   - Replace <DATABASE_NAME> with the name of the database created in previous steps
+
+- Configure Postgres admin username and password following the instructions outlined here: [Using External Database](#using-external-database)
+
+### Database Admin User Password
+We now require administrator access to configure the database correctly. The following section describes how to configure the administrator credentials for both internal and external database configurations.
+
+#### Using Internal Database
```

Comment:
Change internal to On Premise to be consistent.  Update any links to this section